cmd/job-executor-service-initcontainer/main.go

```diff
@@ -112,25 +118,20 @@ func main() {
 		log.Fatalf("unable to create keptn API: %s", err)
 	}
 
-	useLocalFileSystem := false
-
-	// configure keptn options
-	if env.Env == "local" {
-		log.Println("env=local: Running with local filesystem to fetch resources")
-		useLocalFileSystem = true
-	}
-
 	// re-create the event from job-executor-service
-	eventProps := keptn.EventProperties{
-		Project:     env.Project,
-		Stage:       env.Stage,
-		Service:     env.Service,
-		GitCommitID: env.GitCommitID,
```

Comment:
Why was the GitCommitID removed here? If we use the resource-service we should always try to fetch the resource with the provided commit id

Reply:
The `&keptnv2.EventData` does not have a `GitCommitID` field and the value was also not used in the constructor of the `ResourceHandler`. [This comment](https://github.com/keptn-contrib/job-executor-service/pull/351#discussion_r958420056) should provide more information.